### DISCUSSION TITLE
Question about the Submission Quota

### DISCUSSION THREAD

**Discussion by luckyyangrun** [PaddlePaddle/PaddleSpatial#277] [BaiduKDDCup2022]
Will our submission quota be reset on July 16th at 00:01 AM AOE (20:00 Beijing Time tonight)?

**Comment by ramber1836** [comment]
Yes, it will.

**Reply by aptx1231** [reply]
But why was the submission quota refreshed on 2022-07-17 00:00:00 Beijing?

**Reply by aptx1231** [reply]
@ramber1836
